cookbook/routing/extra_information.rst

```diff
@@ -52,12 +52,16 @@ arguments to your controller:
 
         return $collection;
 
-Now, you can access this extra parameter in your controller::
+Now, you can access this extra parameter in your controller, either as an argument (if specified), or through the ``Request`` object::
 
-    public function indexAction($page, $title)
+    use Symfony\Component\HttpFoundation\Request;
+    
+    public function indexAction(Request $request, $page, $title)
     {
         // ...
+        $titleAttribute = $request->attributes->get('title'); // same as $title
+        // ...
     }
 
 As you can see, the ``$title`` variable was never defined inside the route path,
-but you can still access its value from inside your controller.
+but you can still access its value from inside your controller, or from the ``Request`` object's ``attributes`` bag.
```

Comment:
Can you please break the line here after the first word that crosses the 72nd character?